app/app.rb

```diff
@@ -0,0 +1,76 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative './models/peep.rb'
+require_relative './models/user.rb'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'links/index'
+  end
+
+  post '/peeps/new' do
+    Peep.create(content: params[:new_peep], creator: params[:username])
+    redirect to '/peeps'
+  end
+
+  get '/peeps/new' do
```

Comment:
It's often helpful to name templates after the relevant part of the URL.  Using homogenous language for the same concept really aids code readability.